src/compile_constants.hpp.in

```diff
@@ -14,18 +14,16 @@
 #ifndef COMPILE_CONSTANTS_HPP_
 #define COMPILE_CONSTANTS_HPP_
 
-#define NCONS_MAX @MAX_NUMBER_CONSERVED_VARS@
+#define NCONS_MAX (@MAX_NUMBER_CONSERVED_VARS@)
```

Comment:
I don't think this does anything but I guess it's like slightly more safe stylistically? I got paranoid.